include/SDL3/SDL_gpu.h

```diff
@@ -158,38 +158,61 @@ typedef enum SDL_GPUTextureFormat
     SDL_GPU_TEXTUREFORMAT_B5G5R5A1_UNORM,
     SDL_GPU_TEXTUREFORMAT_B4G4R4A4_UNORM,
     SDL_GPU_TEXTUREFORMAT_R10G10B10A2_UNORM,
+    SDL_GPU_TEXTUREFORMAT_R8G8_UNORM,
     SDL_GPU_TEXTUREFORMAT_R16G16_UNORM,
     SDL_GPU_TEXTUREFORMAT_R16G16B16A16_UNORM,
     SDL_GPU_TEXTUREFORMAT_R8_UNORM,
```

Comment:
Can we rearrange this so we have:
```
SDL_GPU_TEXTUREFORMAT_INVALID = -1,

A8_UNORM,
R8_UNORM,
R8G8_UNORM,
R8G8B8A8_UNORM,
R16_UNORM,
R16G16_UNORM,
R16G16B16A16_UNORM,
```
in a row? That way we have a nice R -> RG -> RGBA sequence. We're already doing this for the SNORMs and FLOATs.

The BGRAs can go after that.